### PR TITLE
CSVLintBear: Rename CsvLintBear

### PR DESCRIPTION
**Pull request by stefanbalas**
Closes https://github.com/coala/coala-bears/issues/955

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
typo in name, CSVLintBeaT

File: bears/csv/.coafile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Default]
+bear_dirs = .
+bears = CSVLintBear
+files = t3.csv
+log_level = DEBUG
+flush_cache = True
```
Comment (sils) [inline bears/csv/.coafile]: did you intentionally comit this file?

Comment (stefanbalas) [inline bears/csv/.coafile]: no, i will delelte it right now

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/958/commits/f259aa191dcee0b7db3b542168bd8040e8100467

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/958/commits/f0821732c21ca45f8f2d16d687f6748e22720107

File: bears/csv/CSVLintBear.py [hunk 2]
```diff
@@ -3,9 +3,8 @@
 
 @linter(executable='csvlint',
         output_format='regex',
-        output_regex=r'\d\. (?P<message>.+)\. Row:'
-                     r' (?P<row>\d+)\. (?P<information>.+)',
-        result_message='This ``csv`` file is invalid.')
+        output_regex=r'\d\. (?P<message>.+(s|g|e|w)\.*)(?P<additional_info>('
+                     r' Row: \d+.|$| Column: \d+).*)')
```
Comment (sils) [inline bears/csv/CSVLintBear.py]: why spec row and column if you ignore them completely? Why ignore them completely?

**Comment by adtac** [comment]
ack https://github.com/coala/coala-bears/pull/958/commits/48df42914f329be28b1ffaa79159c9315b2b25c2

**Comment by adtac** [comment]
I don't know enough about csv and that regex to review https://github.com/coala/coala-bears/pull/958/commits/2c9941b2327a184c94427eb3b7802c42c4305281 sorry

**Comment by Adrianzatreanu** [comment]
ack a2f66cb28e069ae54478f1ff9bdf3e75ee00c0cf

**Comment by Adrianzatreanu** [comment]
ack 45ba76051714012d283cf768f918e1e6bc9c0aad

**Comment by Adrianzatreanu** [comment]
ack 0d28ec9042830800d253f6cdc1f3ef05f8716996

**Comment by sils** [comment]
@rultor merge

**Pull request merged**